src/composables/fungibleTokens.ts

```diff
@@ -33,13 +34,23 @@ let composableInitialized = false;
 
 /**
  * List of all fungible tokens available on user's protocols.
+ * Does not include custom tokens extracted from user's token balances.
  * As this list is quite big (hundreds of items) it requires processing optimizations.
  */
 const tokensAvailable = useStorageRef<ProtocolRecord<AssetList>>(
   {},
   STORAGE_KEYS.fungibleTokenList,
 );
 
+/**
+ * List of custom fungible tokens available on user's protocols.
+ * Includes tokens from the user's account that are not on the main list.
+ */
+const customTokensAvailable = useStorageRef<ProtocolRecord<AssetList>>(
+  {},
+  STORAGE_KEYS.customFungibleTokenList,
+);
```

Comment:
Maybe instead of storing this tokens there could be a computed property that takes the token balances and return list of tokens?
This way we wouldn't need any function that sets the state and there will be no need to store the data in the local storage.